src/sentry/services/hybrid_cloud/organization/model.py

```diff
@@ -8,22 +8,36 @@
 from pydantic import Field
 
 from sentry.constants import ObjectStatus
-from sentry.models.organization import OrganizationStatus
-from sentry.models.organizationmember import InviteStatus
 from sentry.roles import team_roles
 from sentry.roles.manager import TeamRole
 from sentry.services.hybrid_cloud import RpcModel
 
 
-def team_status_visible() -> int:
-    from sentry.models import TeamStatus
+class _DefaultEnumHelpers:
+    """Helper functions to avoid importing sentry.models globally"""
```

Comment:
"Non-instantiated class as a semantic grouping of static methods" is a shameless Java idiom, but IMHO it's worth doing here.

Amusingly, because it persuades Black to let me put one blank line between the functions instead of two, we gain back the extra lines of source that are spent on the `@staticmethod` decorators. (Well, I was amused, anyway.)